locations/jclouds/src/main/java/org/apache/brooklyn/location/jclouds/JcloudsLocation.java

```diff
@@ -931,14 +960,23 @@ protected MachineLocation obtainOnce(ConfigBag setup) throws NoMachinesAvailable
                             for (String rule : iptablesRules) {
                                 batch.add(rule);
                                 if (batch.size() == 50) {
-                                    ((SshMachineLocation)machineLocation).execCommands("Inserting iptables rules, 50 command batch", batch);
+                                    executeCommandWarningOnError(
+                                            (SshMachineLocation)machineLocation,
+                                            "Inserting iptables rules, 50 command batch",
+                                            batch);
                                     batch.clear();
                                 }
                             }
                             if (batch.size() > 0) {
-                                ((SshMachineLocation)machineLocation).execCommands("Inserting iptables rules", batch);
+                                executeCommandWarningOnError(
+                                        (SshMachineLocation)machineLocation,
+                                        "Inserting iptables rules",
+                                        batch);
                             }
-                            ((SshMachineLocation)machineLocation).execCommands("List iptables rules", ImmutableList.of(IptablesCommands.listIptablesRule()));
+                            executeCommandWarningOnError(
```

Comment:
On balance I agree with this, but I suspect it will affect some people who have config for `openIptables` irrespective of the various distros. But we can certainly live with that.

We definitely need to remember to put in the release notes for 0.10.0 that we now throw errors on failure.